src/Compiler/FSComp.txt

```diff
@@ -1105,9 +1105,6 @@ lexIfOCaml,"IF-FSHARP/IF-CAML regions are no longer supported"
 1213,tcConditionalAttributeUsage,"Attribute 'System.Diagnostics.ConditionalAttribute' is only valid on methods or attribute classes"
 #1214,monoRegistryBugWorkaround,"Could not determine highest installed .NET framework version from Registry keys, using version 2.0"
 1215,tcMemberOperatorDefinitionInExtrinsic,"Extension members cannot provide operator overloads.  Consider defining the operator as part of the type definition instead."
-1216,ilwriteMDBFileNameCannotBeChangedWarning,"The name of the MDB file must be <assembly-file-name>.mdb. The --pdb option will be ignored."
-1217,ilwriteMDBMemberMissing,"MDB generation failed. Could not find compatible member %s"
-1218,ilwriteErrorCreatingMdb,"Cannot generate MDB debug information. Failed to load the 'MonoSymbolWriter' type from the 'Mono.CompilerServices.SymbolWriter.dll' assembly."
```

Comment:
Is there a way to leave these errors somehow, at least commented out, for future reference?

Reply:
> I don't think so.

Isn't it what `1214` uses above?

> So if anyone is interested, they can always spelunk git.

The errors are currently scattered between a few places, it would probably be good to simplify the errors lookup as much as we could if it's as simple as commenting an error out.

Reply:
Good eyes, it does and I can't really help old stuff, but for sure, now that it has been pointed out, I will go through and clear out any and all commented out old strings.

Commenting out old code and program data, is probably not really the right thing, even if it is easy.

Reply:
Ah, I see. These are really old commented out errors. I thought it could be some more recent things that could be helpful to look at easier.